### PR TITLE
Print rate limiting info

### PR DESCRIPTION
**Pull request by isuruf**
Copied code from `staged-recipes`

### CODE REVIEW

**Comment by jakirkham** [comment]
Maybe we should make the token an argument. Would be useful if we start using multiple.

**Comment by isuruf** [comment]
Then I'd guess we should print info for everything.

**Comment by isuruf** [comment]
Fixed it and pushed to a new branch in this repo

File: conda_forge_webservices/webapp.py [hunk 1]
```diff
@@ -51,7 +51,6 @@ def print_rate_limiting_info():
 
     d = [
          (os.environ['GH_TOKEN'], "conda-forge-linter"),
-         (os.environ['FEEDSTOCKS_GH_TOKEN'], "conda-forge-coordinator"),
```
Comment (jakirkham) [inline conda_forge_webservices/webapp.py]: We still need to push though. 😕

Comment (isuruf) [reply conda_forge_webservices/webapp.py]: Git operations don't count towards GitHub API

Comment (jakirkham) [reply conda_forge_webservices/webapp.py]: Good to know. Must have missed the documentation for that. Though we may start using the API to handle issue ( https://github.com/conda-forge/conda-forge-webservices/issues/121 ).

File: conda_forge_webservices/webapp.py [hunk 2]
```diff
@@ -23,6 +24,42 @@
...
+    gh_api_reset_time = gh.get_rate_limit().rate.reset
+    gh_api_reset_time -= datetime.utcnow()
+    msg = "{user} - remaining {remaining} out of {total}.".format(remaining=gh_api_remaining,
+            total=gh_api_total, user=user)
+    print("-"*len(msg))
+    print(msg)
+    print("Will reset in {time}.".format(time=gh_api_reset_time))
+
+
+def print_rate_limiting_info():
```
Comment (jakirkham) [inline conda_forge_webservices/webapp.py]: Maybe we can just make token and name arguments here too. That way we only need print the one relevant for the different services. This should play more nicely as we add more tokens.

Comment (isuruf) [reply conda_forge_webservices/webapp.py]: This way it's easier to figure out from the heroku logs. Besides the call to check rate limit is not counted towards the limit.

Comment (jakirkham) [reply conda_forge_webservices/webapp.py]: Maybe we should look into logging instead of using `print`, but that is a separate discussion.

**Comment by jakirkham** [comment]
Thanks @isuruf.

**Pull request merged**